contrib/build-openbmc.sh

```diff
@@ -18,5 +18,3 @@ meson setup build-openbmc \
     $@
 
 ninja install -C build-openbmc
-build-openbmc/src/fwupdtool get-devices --verbose
-test $? -eq 2 || exit 1
```

Comment:
why disable it?  shouldn't it be better to fix the issue?

Reply:
I certainly could; I just wondered what the point running fwupdtool was if there were no devices on CI.

Reply:
I guess I don't feel too strongly on that case, just wanted to make sure it's not hiding a bug by not running the tests

Reply:
Okay; point taken -- lets try this version.